docs/wallets.md

```diff
@@ -22,6 +22,19 @@ Supporting seed generation is a little more involved so the tradeoff between
 implementation complexity and user value is less clear, especially since the
 Codex provides users instructions on doing generation themselves.
 
+## Error Detection and Correction
+
+Wallets MUST support detection of errors using the codex32 checksum algorithm.
+Wallets MAY additionally support error correction; such wallets are referred to as "error-correcting wallets (ECWs)" and have additional requirements.
+
+An ECW
+
+* MUST support correction of up to 4 substitution errors;
+* MAY also support correction of up to 8 erasures;
+* MAY support correction of further errors, including insertion or deletion errors.
```

Comment:
We listed the max erasures and errors that may be corrected but still find the closest valid string. Deleting 6 characters and inserting 4 missing characters are near certainly the limit before these edits give valid strings that are not the closest. With the limit likely 3 and 3 on the first string with an unknown correct length, but assuming the correct length is 48, 74 gives the full 6/4 from the 1st string, at the expense of being able to correct invalid (deliberately) unusual length 1st strings that are too near 48 and 74.
Any reason to not mention these limit? With 13 insertions, everything has a valid correction candidate (append checksum) but it's extremely unlikely to be the closest or intended.

Reply:
Because you can only correct insertions or deletions via brute-force, and the result is never guaranteed to be unique, so there are no limits that make sense.

Reply:
Is there a quantity of inserts or deletes that is guaranteed to NOT be unique? (besides the trivial 13 inserts)

Do we need to specify precedence between substitutions and the optional insertion/deletions? If a string becomes valid with 4 substitutions as well as 1 delete and 1 insert (and they are NOT the same correction), should we say which should be displayed?